Dockerfile.reborn

```
@@ -60,6 +60,12 @@ RUN cargo chef cook \
     --package ironclaw_reborn_cli \
     --features webui-v2-beta,slack-v2-host-beta,libsql,postgres,inmemory-turn-state \
     --recipe-path recipe.json
+RUN cargo chef cook \
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)